src/test/java/com/exasol/bucketfs/SyncAwareBucketIT.java

```diff
@@ -64,62 +43,80 @@ private Path createTestFile(final Path tempDir, final String fileName, final int
     }
 
     // [itest->dsn~uploading-to-bucket~1]
-    @ValueSource(strings = { "dir1/", "dir2/sub2/", "dir3/sub3/subsub3/", "/dir4/", "/dir5/sub5/" })
     @ParameterizedTest
-    void testUploadToDirectoryInBucket(final String pathInBucket, @TempDir final Path tempDir)
-            throws BucketAccessException, InterruptedException, IOException, TimeoutException {
+    @ValueSource(strings = { "dir1/", "dir2/sub2/", "dir3/sub3/subsub3/", "/dir4/", "/dir5/sub5/" })
+    void testUploadToDirectoryInBucket(final String pathInBucket, @TempDir final Path tempDir) throws Exception {
         final var fileName = "file.txt";
         final var file = createTestFile(tempDir, fileName, 1);
-        final var bucket = createDefaultBucket();
+        final var bucket = getDefaultBucketForWriting();
         bucket.uploadFile(file, pathInBucket);
-        assertThat(createDefaultBucket().listContents(pathInBucket), Matchers.contains(fileName));
+        assertThat(getDefaultBucketForWriting().listContents(pathInBucket), Matchers.contains(fileName));
     }
 
     // [itest->dsn~uploading-strings-to-bucket~1]
     @Test
-    void testUploadStringContent() throws IOException, BucketAccessException, InterruptedException, TimeoutException {
+    void testUploadStringContent() throws Exception {
```

Comment:
Are you sure, Sonar is OK with this contraction of exceptions? Because I am skeptical. Sure, this is more compact, but it also hides what are "valid" exceptions that can be thrown here.

Reply:
Rationale for my changes 
* increased readability, less clutter, less code 
* no loss in functionality, expressiveness, no increased risk for unidentified bugs as test is intended to not throw an exception anyway